tests/nv_ingest_client/primitives/tasks/test_store.py

```diff
@@ -29,7 +32,14 @@ def test_store_task_initialization():
 
 
 def test_store_task_str_representation():
-    task = StoreTask(structured=True, images=True, store_method="minio", endpoint="minio:9000")
+    task = StoreTask(
+        structured=True, 
+        images=True, 
+        store_method="minio", 
+        extra_params={
```

Comment:
these test edits are related to the previously `extra_params` changes in this review.